src/kaltura-live-plugin.tsx

```diff
@@ -250,6 +245,7 @@ export class KalturaLivePlugin implements OnMediaUnload, OnPluginDestroy {
 
         if (receivedState === LiveBroadcastStates.Offline) {
             this._manageOfflineSlate(OfflineTypes.NoLongerLive);
+            this._player.dispatchEvent(new (KalturaPlayer.core as any).FakeEvent(this._player.Event.ENDED, { manualEvent: true }));
```

Comment:
Why are we firing ended on our own here and not actually letting player end stream and fire on its own?
What is manualEvent: true?

Reply:
plugin prevents "END" event in decorator. "END" event should be fired only after plugin get response from BE